### PR TITLE
Feat/exit multiple tx hashes

### PR DESCRIPTION
**Pull request by shanefontaine**
@miguelmota Only downside of this is that it fires off a tx per exit. However, I think the simplicity of that (as opposed to trying to batch many exits in a single tx) is worth it.

### CODE REVIEW

File: packages/hop-node/src/cli/exitCommitTx.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ root
   .description('Exit the commit transaction')
   .option('--chain <slug>', 'Chain', parseString)
   .option('--token <symbol>', 'Token', parseString)
-  .option('--tx-hash <hash>', 'Tx hash with CommitTransfers event log', parseString)
+  .option('--tx-hashes <hash>', 'Comma-separated tx hashes with CommitTransfers event log', parseString)
```
Comment (miguelmota) [inline packages/hop-node/src/cli/exitCommitTx.ts:15]: current

```
<hash>
```

suggestion

```
<hash, ...>
```

---

current

```ts
parseString
```

suggestion

```ts
parseStringArray
```

and add this to `shared.ts`

```ts
export function parseStringArray (value: string) {
  return (value ?? '').split(',').map((v: string) => v.trim())
}
```

Comment (miguelmota) [reply packages/hop-node/src/cli/exitCommitTx.ts:15]: @shanefontaine

File: packages/hop-node/src/cli/exitCommitTx.ts [hunk 2]
```diff
@@ -38,6 +38,9 @@ async function main (source: any) {
   }
 
   const chainSpecificWatcher: ExitWatcher = watcher.watchers[chain]
-  await chainSpecificWatcher.relayXDomainMessage(commitTxHash)
+  const txHashes = commitTxHashes.split(',')
```
Comment (miguelmota) [inline packages/hop-node/src/cli/exitCommitTx.ts:41]: with above suggestion, `commitTxHashes` would be an array

File: packages/hop-node/src/cli/exitCommitTx.ts (hunk 1 above)
Comment (shanefontaine) [reply packages/hop-node/src/cli/exitCommitTx.ts:15]: Resolved in 62d3ebb08a4d703cac759d6ab80f7d9dfdabfeac

File: packages/hop-node/src/cli/exitCommitTx.ts (hunk 2 above)
Comment (shanefontaine) [reply packages/hop-node/src/cli/exitCommitTx.ts:41]: Resolved in 9ee3ae58c72894d53f400a51afb8db5cc4f721f8

**Pull request merged**